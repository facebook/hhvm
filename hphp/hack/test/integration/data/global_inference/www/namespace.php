<?hh //partial

namespace Foo\Bar;

class MyClass {
}

function buildermyclass() {
  return new MyClass();
}
