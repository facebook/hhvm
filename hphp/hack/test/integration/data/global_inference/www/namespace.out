<?hh //partial

namespace Foo\Bar;

class MyClass {
}

function buildermyclass() : <<__Soft>> \Foo\Bar\MyClass {
  return new MyClass();
}
